libpod/image/image.go

```diff
@@ -1357,60 +1300,12 @@ func (i *Image) GetChildren(ctx context.Context) ([]string, error) {
 
 // getChildren returns a list of at most "max" imageIDs that depend on the image
 func (i *Image) getChildren(ctx context.Context, max int) ([]string, error) {
```

Comment:
(I’d prefer the caller, incl. the caller of the higher-level `GetChildren` to have to supply the `layerTree` to encourage caching it over multiple calls.)